docs/quick_start/README.md

```diff
@@ -0,0 +1,54 @@
+# Quickstart
+
+**Before you start:** Make sure you've [installed Janus](../install/README.md)
+
+In this section, you'll learn how to manage your Janus instance. First we'll
+have you start Janus giving in order to give you access to the RESTful Admin
+interface, through which you manage your APIs, consumers, and more. Data sent
+through the Admin API is stored in Janus's [datastore](../datastore/README.md) (Janus
+supports File System and MongoDB).
+
+1. ### Start Janus.
+
+    We recommend using some sort of process control, like [runit](http://smarden.org/runit).
+    
+    Issue the following command to [start][CLI] Janus:
+
+    ```bash
+    $ sv start janus
+    ```
+
+    **Note:** The CLI also accepts a configuration (`-c <path_to_config>`)
```

Comment:
This is one is something that I want to implement there, but we still don't have it. I'll remove it for now